data/lm/README.rst

```diff
@@ -1,32 +1,23 @@
-Download librispeech corpus
+The LM binary was generated from the LibriSpeech normalized LM training text, available `here <http://www.openslr.org/11>`_.
+It is created with `KenLM <https://github.com/kpu/kenlm>`_.
+
+You can download the librispeech corpus with the following commands:
 
 .. code-block:: bash
 
     wget http://www.openslr.org/resources/11/librispeech-lm-norm.txt.gz -O librispeech.txt.gz
     gunzip librispeech.txt.gz
 
-|
-| Generate vocab-500000.txt and lm.binary files
-| Optional Parameters:
-
-* '--kenlm_bins path/to/bins/':  Change the path of the kenlm binaries (defaults to directory in docker container)
-* '--top_k 500000': Change the number of most frequent words
-* '--arpa_order 5': Change order of k-grams in arpa-file generation
-* '--max_arpa_memory 75%': Set maximum allowed memory usage for arpa-file generation
 
+Then use the `generate_lm.py` script to generate `lm.binary` and `vocab-500000.txt`.
 
 .. code-block:: bash
 
     python3 data/lm/generate_lm.py --input_txt path/to/vocab_sentences.txt --output_dir path/lm/
```

Comment:
This command should fully specify all the parameters matching the included LM.

Reply:
And since `generate_lm.py` now takes a parameter with the path to KenLM tools instead of relying on `$PATH`, this README should at least link to the KenLM build documentation and say that those steps need to be followed first.

Reply:
I still don't like the use of the parameters without default values. I understand we should specify all the parameters matching the included LM somewhere, but why not as default values? 
At least some of the lesser changed values like `-arpa_prune` or `--binary_type`. 

It would keep the command with specific changes much more readable:
`python3 data/lm/generate_lm.py --input_txt path/to/vocab_sentences.txt --output_dir path/lm/ --arpa_order 3`
Instead of:
`python3 data/lm/generate_lm.py --input_txt path/to/vocab_sentences.txt --output_dir path/lm/ -top_k 500000 --kenlm_bins /DeepSpeech/native_client/kenlm/build/bin/ --arpa_order 3 --max_arpa_memory "75%" --arpa_prune "0|0|1" --binary_a_bits 255 --binary_q_bits 8 --binary_type trie`
In the above command you can see on the first look that i only changed the arpa order.

Can we ask @lissyx for opinion?

Reply:
For example, if you call the script with with `--arpa_order 2` you also have to modify the default pruning parameter. If we include defaults, people will run this script against their own data, modify some parameter, then the other defaults will conflict, and then they'll file an issue on our repo.

Besides, there's also the fact that all the default values included here are arbitrary, and only make sense for the default LM. There is no reason these values should be a default for any text corpus, so we should not misguide people using these scripts.

Reply:
Thanks for your explanation.